docs/topics/api/reviewers.rst

```diff
@@ -349,7 +349,7 @@ These endpoints allow you to draft comments that can be submitted through the re
     Create a draft comment for a specific version.
 
     :<json string comment: The comment that is being drafted as part of a review.
-    :<json string filename: The filename this comment is related to (optional).
+    :<json string filename: The absolute file path this comment is related to (optional). This must represent the absolute path, including sub-folders.
```

Comment:
I think it's misleading to call this an absolute path. It's really a full path, relative to the root of the add-on. For example, `background.js` or `lib/scripts/background.js` -- I wouldn't consider either of those an absolute path.

Reply:
I tried to clarify this in eeb8317, maybe that's a bit better?